docs/azure_arc_jumpstart/azure_arc_data/kubeadm/_index.md

```diff
@@ -1,8 +1,8 @@
 ---
 type: docs
-title: "Upstream Kubernetes (Kubeadm)"
-linkTitle: "Upstream Kubernetes (Kubeadm)"
-weight: 7
+title: "Kubernetes Kubeadm"
+linkTitle: "Kubernetes Kubeadm"
+weight: 2
```

Comment:
```suggestion
weight: 7
```

Reply:
We still want kubeadm to be in that position 

![image](https://user-images.githubusercontent.com/46730876/193034601-cf9f5dd6-fc72-4d44-97c5-3b6508b99fc6.png)